src/components/StylesFromLiteralsService.js

```diff
@@ -0,0 +1,154 @@
+(function() {
+  goog.provide('ga_styles_from_literals_service');
+
+  var module = angular.module('ga_styles_from_literals_service', []);
+
+  module.provider('gaStylesFromLiterals', function() {
+
+    this.$get = function() {
+
+      function getOlStyleForPoint(options, shape) {
+        if (shape === 'circle') {
+          return new ol.style.Circle(options);
+        } else if (shape === 'icon') {
+          return new ol.style.Icon(options);
+        } else {
+          var shapes = {
+            square: {
```

Comment:
We need all the basic geometric forms. Circle, triangle, square, 5 branches star etc..

What about MapFish print server version 3? Those should be compatible.